### ISSUE TITLE
`UpsertTest::testUpsertWithMultipleSet` fails

### ISSUE THREAD

**Issue by kenjis** [codeigniter4/CodeIgniter4#6691]
Related #6600
See also https://github.com/codeigniter4/CodeIgniter4/pull/6690#issuecomment-1279611246

```
There was 1 failure:

1) CodeIgniter\Database\Live\UpsertTest::testUpsertWithMultipleSet
Failed asserting that a row in the table [user] matches the attributes 
{
    "email": "jarvis@example.com",
    "name": "Jarvis",
    "country": "2022-10-14"
}

Found similar results: [
    {
        "id": "5",
        "name": "Jarvis",
        "email": "jarvis@example.com",
        "country": "2022-10-15",
        "created_at": null,
        "updated_at": null,
        "deleted_at": null
    }
] and 2 others.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Test/DatabaseTestTrait.php:280
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Database/Live/UpsertTest.php:641
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:97
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/3253661191/jobs/5341121215

**Comment by sclubricants** [comment]
https://stackoverflow.com/questions/381371/sqlite-current-timestamp-is-in-gmt-not-the-timezone-of-the-machine

**Comment by sclubricants** [comment]
I'll put a fix in in a minute

**Comment by sclubricants** [comment]
https://github.com/codeigniter4/CodeIgniter4/pull/6692

**Comment by kenjis** [comment]
Closed by #6692
